alf/utils/dist_utils.py

```diff
@@ -69,27 +69,27 @@ def _call(self, x):
 
     def _inverse(self, y):
         """
-        y > 0:
-            y = x / (1+x) => x = y / (1 - y)
-        y < 0:
-            y = x / (1-x) => x = y / (1 + y)
+        :math:`y > 0`:
+            :math:`y = \frac{x}{1+x}\Rightarrow x = \frac{y}{1 - y}`
+        :math:`y < 0`:
+            :math:`y = \frac{x}{1-x}\Rightarrow x = \frac{y}{1 + y}`
         """
         return torch.where(y > 0, y / (1 - y), y / (1 + y))
 
     def log_abs_det_jacobian(self, x, y):
         """
-        x > 0:
-            y = x / (1+x) => dy/dx = -1/(1+x)^2
-        x < 0:
-            y = x / (1-x) => dy/dx = 1/(1-x)^2
+        :math:`x > 0`:
+            :math:`y = \frac{x}{1+x}\Rightarrow \frac{dy}{dx} = \frac{-1}{(1+x)^2}`
+        :math:`x < 0`:
+            :math:`y = \frac{x}{1-x}\Rightarrow \frac{dy}{dx} = \frac{1}{(1-x)^2}`
         """
         return -2. * torch.log(1 + x.abs())
 
 
 @gin.configurable
 class StableTanh(td.Transform):
-    """Invertable transformation (bijector) that computes `Y = tanh(X)`,
-    therefore `Y in (-1, 1)`.
+    """Invertable transformation (bijector) that computes :math:`Y = tanh(X)`,
+    therefore :math:`Y \in (-1, 1)`.
 
     This can be achieved by an affine transform of the Sigmoid transformation,
     i.e., it is equivalent to applying a list of transformations sequentially:
```

Comment:
`.. code-block:: python` for the code block below

Reply:
updated